tests/schema/tests.py

```diff
@@ -1956,6 +1956,21 @@ def test_alter_field_default_dropped(self):
             if connection.features.can_introspect_default:
                 self.assertIsNone(field.default)
 
+    @unittest.skipIf(connection.vendor == 'sqlite', 'SQLite naively remake table on field alteration.')
+    def test_alter_field_default_doesnt_perfom_queries(self):
+        """
+        No queries should be performed if a field default changes unless a
```

Comment:
I guess I'd say, "No queries are performed if a field default changes and the field's not changing from null to non-null." 
The "unless" phrasing may give the idea that case is also tested here.